docs/guide/security-conventions.md

```diff
@@ -0,0 +1,23 @@
+This section provide instructions to comply with the standard convention how to secure code in development.
+
+## For Code: 
+### 1. Validate input. 
+Validate input from all untrusted data sources. Proper input validation can eliminate the vast majority of software vulnerabilities. Be suspicious of most external data sources, including command line arguments, network interfaces, environmental variables, and user controlled files [Seacord 05]. 
```

Comment:
[Seacord 05]. <- delete this part please

Reply:
done..